packages/serverpod/lib/src/server/server.dart

```diff
@@ -205,6 +215,21 @@ class Server {
       readBody = false;
     }
 
+    // This OPTIONS check is necessary when making requests from
+    // eg `editor.swagger.io`. It ensures proper handling of preflight requests
+    // with the OPTIONS method.
+    if (request.method == 'OPTIONS') {
+      for (var header in httpOptionsResponseHeaders.entries) {
+        request.response.headers.add(header.key, header.value);
+      }
+
+      // Safari and other potential browsers required to set content-length=0
```

Comment:
Spelling misstake here:
`// Safari and potentially other browsers require Content-Length=0.`